combat-trainer.lic

```diff
@@ -296,8 +295,7 @@ class SetupProcess
   end
 
   def determine_whirlwind_weapon(game_state)
-    temp_array = (game_state.whirlwind_trainables - [game_state.weapon_skill])
-    offhand_skill = temp_array.min_by { |skill| [DRSkill.getxp(skill), DRSkill.getrank(skill)] }
+    offhand_skill = game_state.sort_by_rate_then_rank(game_state.whirlwind_trainables - [game_state.weapon_skill]).first
```

Comment:
Original logic already sorted by rate then rank, this change is to make that decision consistent throughout combat-trainer by using `game_state.sort_by_rate_then_rank`. _If_ the decision algoritm should change in the future, there's now one spot to do it.